android/java/res/layout/tx_details_bottom_sheet.xml

```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2023 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
```

Comment:
Is it not possible to put the comments outside the XML blocks? Seems odd that these aren't at the top here but I'm assuming that this is a normal convention I'm just noticing now.

Reply:
Moving this at top isn't possible, it causes compile time errors so that's why it has to be below xml declaration.